### DISCUSSION TITLE
How to contribute to Keploy and join on slack?

### DISCUSSION THREAD

**Discussion by yaten2302** [keploy/keploy#1979] [Q&A] [answered]
Hey @keploy team!!

I'm Yaten,  I've been contributing to open source a lot and I'm an active contributor at `Taipy` as well. I've knowledge of ReactJS, TypeScript, JavaScript, but I've knowledge of Go at very basic level(just started learning it).
I'm very keen to contribute to Keploy and improve my Go skills. Can anyone please guide me that how can I learn Go and start contributing to keploy?

Also, can anyone please guide that can new contributors also join keply on slack? Because, I don't have a `@keploy.io`. Is this only for the maintainers or new contributors can also join?

Thanks

**Comment by Sonichigo** [comment, marked as answer]
Hey @yaten2302 ! Welcome to the Keploy Community! 

You can try out our [samples-go](https://github.com/keploy/samples-go) application to get started with Go. Learn how the routers works and which best suits for you, and then run that application with Keploy to see with debuggers to understand how the Keploy works. 

Once you get better understanding we have some good first issues [here](https://github.com/keploy/keploy/issues?q=is%3Aissue+is%3Aopen+label%3A%22Good+First+Issue%22) which you can pick and try out. 

Keploy Slack is open for all to join: - https://join.slack.com/t/keploy/shared_invite/zt-2dno1yetd-Ec3el~tTwHYIHgGI0jPe7A 

Do let me know if you face any issue or need help!  Glad to have you here!!

**Comment by yaten2302** [comment]
Hey @Sonichigo, thank you so much for the warm welcome to Keploy :)
Thanks for your suggestions, I'll surely check the Go samples given above and as soon as I'm familiar with Go, I'll start contributing to Keploy 👍 
I joined the slack as well, thanks for the link 👍

If I face any issue, I'll surely ask it in the slack or GitHub discussions 👍

**Comment by yaten2302** [comment]
Hi @Sonichigo, actually, I've just started learning Go, could you please guide with some docs or YouTube channel for learning it?

**Reply by Sonichigo** [reply]
I would recommend to follow go playground: - https://go.dev/play/  and freeCodeCamp YT guide :-  https://www.youtube.com/watch?v=un6ZyFkqFKo

**Reply by yaten2302** [reply]
Got it, thank you so much @Sonichigo 👍
